packages/head/index.tsx

```diff
@@ -1,9 +1,34 @@
 import Head from 'next/head'
+import isAbsoluteUrl from './helpers/is-absolute-url'
 import { renderMetaTags } from './seo'
 
 export { renderMetaTags }
 
+const IS_DEV = process.env.NODE_ENV !== 'production'
+
 export default function HashiHead(props: HashiHeadProps): React.ReactElement {
+  /**
+   * Throw an error if props.image is a relative URL.
+   * It must be an absolute URL in order to work as expected as og:image.
+   * Reference: https://ogp.me/#url
+   */
+  if (typeof props.image !== 'undefined' && !isAbsoluteUrl(props.image)) {
+    /**
+     * TODO: should we consider alternatives to throwing an error here?
```

Comment:
@BRKalow Forget to mention in review - wanted to get your take on the error approach here. I think the `IS_DEV` approach could make sense, but would love a second opinion on that (and also a lil curious about whether further tracking, eg in DataDog if available, sounds like something we might want to do).

Reply:
If our goal is to fix all existing now-incorrect values for `props.image,` it seems reasonable to me to always throw an error regardless of the `NODE_ENV`. No production builds will be immediately broken unless the version for this package is upgraded after release right? We'll just start seeing failed Vercel builds in PRs?

Reply:
☝️ I think we'd definitely want to always throw an error if we did 100% static builds, but might not be as safe anymore since we don't static render all pages... for ISR pages, I think throwing an error could cause a client-side error, or possibly a 500 for the initial server-side render? Does that sound accurate?

Reply:
Hmm, yeah. That makes sense. I wonder if we can test different cases to make sure? But that can definitely be done outside of this PR! We can always update this block in the future if needed. 📌